lib/gui/app.js

```diff
@@ -134,7 +156,7 @@ app.run(($timeout, DriveScannerService, DrivesModel, ErrorService, DriveSelector
   DriveScannerService.on('drives', (drives) => {
 
     // Safely trigger a digest cycle.
-    // In some cases, AngularJS doesn't aknowledge that the
+    // In some cases, AngularJS doesn't acknowledge that the
```

Comment:
Created https://github.com/resin-io/etcher/pull/1145. I also fixed the typo in that PR, will remove it from this one.